src/Tools/babylon.assetsManager.ts

```diff
@@ -263,6 +263,12 @@
         public onTaskSuccess: (task: IAssetTask) => void;
         public onTaskError: (task: IAssetTask) => void;
 
+        //Observables
+
+        public onTaskSuccessObservable = new Observable<IAssetTask>();

```

Comment:
I would have added some on each task as well :)

Reply:
Would that make sense? since we register tasks only with the assets manager?

Reply:
yeah makes sense..It was a test, you passed!

Reply:
I want an observing-observer certificate.